### ISSUE TITLE
What is your opinion of the Bible?

### ISSUE THREAD

**Issue by dandv** [IonicaBizau/ama#16]
Pardon the [SPR](https://www.glassdoor.com/blog/sex-religion-politics-stay-discussing-office/) question, but since you've brought up the Bible here on GitHub, may we ask about your opinion of it?
- How historically and scientifically accurate do you think it is, and what makes you believe that?
- Do you believe it is the word of God, or fables/legends/moral precepts recorded by man, or a combination of the two?
- How useful is it to society X thousand years after it was written, to understand the world around us and guide our morality?
- How is it influencing your life?

**Comment by IonicaBizau** [comment]
> How historically and scientifically accurate do you think it is, and what makes you believe that?

I think it's pretty accurate. There are arguments on both sides, but I think the Bible religion doesn't conflict with the true science. 🔬 

I do believe there are mistakes in there, but I don't think they are that important to affect our salvation.

For instance, the battle stats from `2 Samuel 10:18` are different than the ones from `1 Chronicles 19:18` (both verses mention the same battle). Personally I'm not affected by that difference.

One who searches errors in the Bible will find them. It's more constructive to see the whole image the Bible presents us, than taking random verses and finding contradictions.

> Do you believe it is the word of God, or fables/legends/moral precepts recorded by man, or a combination of the two?

I do believe the Bible was written by humans, but it presents the message of God for us.

It does contain parables which made it easier to to explain some points of view: Jesus did use many parables. For example, I don't believe the story presented in Luke 16:19-31 was a real one, but the point there was that Jesus was trying to make people aware about the importance of the previous writings.

> How useful is it to society X thousand years after it was written, to understand the world around us and guide our morality?

Since I believe God is our designer, I think He's the best one to recommend us something to (not) do. One way He talks to us is by the Bible message. 📖 

God didn't made us robots but He gave us the freedom of will. Therefore, humans can use their intelligence in the bad direction. And they pretty much do that. 💣💥

We are not to judge people around us but help them. It's God who knows what's in the heart of each one of us—things we don't see.

> How is it influencing your life?

Pretty much see #17. I read the Bible few times and continue studying it and still finding new things and curiosities. It's like finding new stuff you can do in a programming language (JavaScript in my case) even if you've been working with it for years. 🚀 

----

In conclusion, I believe Bible is a very important book to read—there are many things to learn from there.

Obviously, I'm not against reading any other books (personally I enjoy physics/chemestry-related books too), but I consider Bible more important than any other books because it shows the history of the world since its beginning to its end.

**Comment by Kogam22** [comment]
@IonicaBizau 
Hi There :wave: 

Well, As You Did Research Bible, I'd Like To Ask Whether You Researched Some Other Religions Too.
For Example, Consider Judaism Or Islam. I Particularly Mentioned These Two Because The Bible Mentions About Moses And The Qur'aan Mentions About Jesus And These Books Came Chronologically.

We Also See Many Things That Show Some Sort Of Relation Between The Old Testament, New Testament And The Qur'aan.

Not That I'm A Scholar, But I Watch Debates Often And These Are Things Muslims Say.

**Comment by dandv** [comment]
@Kogam22: what's up with the gratuitous Title Capitalization? Do you realize it only detracts from the credibility of your already dubious post?

@IonicaBizau:

> I do believe the Bible was written by humans, but it presents the message of God for us.

Why do you believe the Bible, written by humans, presents the message of God? Why don't you believe that some other old text, e.g. the Code of Hammurabi, presents the message of God? Is it not possible that those humans have made things up along the way, combining historical events of eroding accuracy as they have been passed along through generations via word of mouth, with tales and myth? After all, there are tens (hundreds?) of [miraculous birth stories](https://en.wikipedia.org/wiki/Miraculous_births). How many have you compared before picking the Jesus one as the true one?

> Since I believe God is our designer,

What makes you believe that, and what evidence would make you stop believing that?

> I think He's the best one to recommend us something to (not) do.

Then why have we as a society decided that many of those recommendations no longer make sense? Slavery, homophobia, stoning people to death etc?

> One way He talks to us is by the Bible message.

Again, how do you know that?

**Comment by IonicaBizau** [comment]
Sorry for replying to this so late, but these months I was involved in other projects and had less free time. 

@Kogam22 I read parts of the Qur'aan, and reading it entirely is on my list too. I like studying how religions developed and compare them. Also, recently I was checking what the Talmud says on specific subjects, and I found it interesting.

@dandv Perhaps the personal experience is one of the reasons why I believe in God and I find the Bible's message a good one for my life. It helps me and keeps me growing continuously. I know people who were strongly against the Bible and, to my joy, at some point they decided to study it with an open heart and eventually start liking it––and the good change was clear to everybody.

I love astronomy and I was always impressed by the precision the position of the stars and other planets can be calculated and indicated on the sky. I refuse believing the Universe, the beauty of sunrises and sunsets, the life and other stuff we have access to, is just a result of billions of years of evolution. To me, personally, believing in a Designer of these, makes way more sense.

Regarding *slavery, homophobia, stoning people to death etc*, I suggest taking the Bible message as a whole, and you will see God condemns such acts. 

Like I mentioned in #37, these are common debates between believers and non-believers and if we start one of these here perhaps we can continue it forever. This happens everywhere (inside of Christianity itself, inside of every church, in tech etc).

I guess we should focus on things that unite us and help other people.
Otherwise, we will always find things which divide us and waste our times.

**Comment by dandv** [comment]
I appreciate your choice to not engage in a less-than-productive discussion. I won't be expecting a reply; just wanted to address, very briefly, some key points.

> I refuse believing the Universe, the beauty of sunrises and sunsets, the life and other stuff we have access to, is just a result of billions of years of evolution. To me, personally, believing in a Designer of these, makes way more sense.

How did the Designer come to be?

![Occam's razor against God](https://i.imgur.com/bmHWg.gif)

> Regarding slavery, homophobia, stoning people to death etc, I suggest taking the Bible message as a whole, and you will see God condemns such acts.

Maybe such acts are condemned in some parts, but they are very much condoned in others. See http://www.evilbible.com, [Bible passages considered immoral by many of today's theologians and secularists](http://www.religioustolerance.org/imm_bibl.htm) and Indicators 3 and 4 from [ReligiousTolerance.org](http://www.religioustolerance.org/ine_none1.htm).

Sure, apologets can weasel their way out of each of these ("incorrect translation", "overruled by some other verse" etc.), but I personally expected more from a book allegedly inspired by an infinite intelligence, than to need advocacy from humans; why not write it clearly in Latin in the first place, and [without the plot holes and nasty stuff](https://skepticsannotatedbible.com/)? Throw in some predictions verifiable a few thousand years later, like the 1,000,000th digit of Pi, and I'd be a lot more convinced it wasn't just a collection of the myths at the time.
